design-system/uilib/src/components/dpdhl-toggle/dpdhl-toggle.stories.svelte

```diff
@@ -0,0 +1,28 @@
+<script lang="ts">
+	import { Meta, Template, Story } from "@storybook/addon-svelte-csf";
+	import "@sprinteins/dpdhl-uilib";
+
+	$: checked = true;
```

Comment:
This can be `let checked = true`
No need for the `$`. we would need the `$` if the right side would change but `true` does not change